x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/index.tsx

```diff
@@ -0,0 +1,144 @@
+import defaultImgSrc from '../../../assets/service_tab_empty_state/service_tab_empty_state_default.png';
+
+export interface ServiceTabEmptyStateProps {
+  title: string;
+  content: string;
+  imgSrc: string | null;
+  dismissable?: boolean;
+  onDissmiss?: () => void;
+}
+
+const addAPMLinkButton = {
+  label: i18n.translate('xpack.apm.serviceTabEmptyState.addAPMButtonLabel', {
+    defaultMessage: 'Add APM',
+  }),
+  href: '/app/observabilityOnboarding/?category=apm',
+};
+
+const tryItNowButton = {
+  label: i18n.translate('xpack.apm.serviceTabEmptyState.tryItNowButtonLabel', {
+    defaultMessage: 'Try it now in our demo cluster',
+  }),
+  href: 'https://ela.st/demo-apm-try-it',
+};
+
+const learnMoreLink = {
+  label: i18n.translate('xpack.apm..serviceTabEmptyState.learnMoreLinkLabel', {
+    defaultMessage: 'Learn more',
+  }),
+  href: 'https://www.elastic.co/observability/application-performance-monitoring',
+};
+
+export function ServiceTabEmptyState({
+  title,
+  content,
+  imgSrc,
+  dismissable,
+  onDissmiss,
+}: ServiceTabEmptyStateProps) {
+  const { core } = useApmPluginContext();
+  const { basePath } = core.http;
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <>
+      <EuiPanel color="subdued" paddingSize="xl" style={{ position: 'relative' }}>
+        <EuiFlexGroup alignItems="center">
+          <EuiFlexItem>
+            <EuiTitle size="s">
+              <h3>{title}</h3>
+            </EuiTitle>
+            <EuiSpacer size="s" />
+            <EuiText size="s">{content}</EuiText>
+            <EuiSpacer size="m" />
+            <EuiFlexGroup alignItems="center" gutterSize="m">
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  data-test-subj="ServiceTabEmptyStateAddApmButton"
+                  href={basePath.prepend(addAPMLinkButton.href)}
+                  fill
+                >
+                  {addAPMLinkButton.label}
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  data-test-subj="ServiceTabEmptyStateTryItNowButton"
+                  iconType="launch"
+                  iconSide="right"
+                  href={tryItNowButton.href}
+                  target="_blank"
+                >
+                  {tryItNowButton.label}
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiLink
+                  href={learnMoreLink.href}
+                  target="_blank"
+                  data-test-subj="infraHostsTableWithoutSystemMetricsPopoverTroubleshootingLink"
+                  external
+                >
+                  {learnMoreLink.label}
+                </EuiLink>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          {!imgSrc && (
+            <EuiFlexItem
+              style={{
+                maxHeight: `${euiTheme.base * 14}px`,
```

Comment:
This is how it looks on my wide screen. it shows just the tabs but we load the entire image 

![image](https://github.com/user-attachments/assets/27d4f603-9cf5-42b3-8027-3d2f336a2b97)

I couldn't find how it supposed to look in the designs.  Is this expected?

Reply:
We decided not to handle responsiveness or different image sizes, even though this might make some images look a bit off. I’ll double check with product and design to see if they want to change the overview image or keep it as it is. 

I’ll let you know once I have their feedback. 
Thank you!

Reply:
Hey @kpatticha,

After discussing with product and design, we've agreed to leave it as it is (with a small height increase), knowing that it won't display perfectly for all users. The image will now be of the service map instead of the overview.

We're open to modifying it in the future with a better approach.

Approximately 16% of users monitored via Fullstory use a `2560px` resolution (and only 4% use >`3000px`), and this is how it will look:

![Screenshot 2024-09-06 at 13 31 35](https://github.com/user-attachments/assets/4832ffe5-b695-413e-ba11-f7f00c311af7)

Reply:
thanks for the update 🔥